### PR TITLE
added two titles with  corresponding links and location of tools

### PR DESCRIPTION
**Pull request by RAJU2529**
i added  the following tools 

 **Recovery Drive  with website link*&
 **Registry Editor with website link**

added  the  sentences **Below tools are located under C:\Windows\Sytem32\**

### CODE REVIEW

File: windows/client-management/administrative-tools-in-windows-10.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The tools in the folder might vary depending on which edition of Windows you are
 
 ![Screenshot of folder of admin tools](images/admin-tools-folder.png)
 
-These tools were included in previous versions of Windows and the associated documentation for each tool should help you use these tools in Windows 10. The following list links to documentation for each tool.
+These tools were included in previous versions of Windows and the associated documentation for each tool should help you use these tools in Windows 10. The following list links to documentation for each tool.Below tools are located under C:\Windows\Sytem32\
```
Comment (illfated) [inline windows/client-management/administrative-tools-in-windows-10.md]:
```suggestion
These tools were included in previous versions of Windows and the associated documentation for each tool should help you use these tools in Windows 10. The following list links to documentation for each tool. The tools are located within the folder C:\Windows\System32\ or its subfolders.
```
This is of course my interpretation, although I have not researched the exact location of each of the 19 tools in the list. I recommend inviting @JohanFreelancer9 to review this PR content and suggest any improvements to it.

**Comment by illfated** [comment]
(for reference: issue ticket #5831 )

**Comment by illfated** [comment]
(waiting for feedback from @JohanFreelancer9 to see if there is room for more improvements)

File: windows/client-management/administrative-tools-in-windows-10.md [hunk 2]
```diff
@@ -43,6 +43,8 @@ These tools were included in previous versions of Windows and the associated doc
 -   [ODBC Data Sources]( https://go.microsoft.com/fwlink/p/?LinkId=708494)
 -   [Performance Monitor](https://go.microsoft.com/fwlink/p/?LinkId=708495)
 -   [Print Management](https://go.microsoft.com/fwlink/p/?LinkId=708496)
+-   [Recovery Drive](https://support.microsoft.com/en-us/help/4026852/windows-create-a-recovery-drive)
```
Comment (JohanFreelancer9) [inline windows/client-management/administrative-tools-in-windows-10.md]:
```suggestion
-   [Recovery Drive](https://support.microsoft.com/help/4026852/windows-create-a-recovery-drive)
```

File: windows/client-management/administrative-tools-in-windows-10.md [hunk 3]
```diff
@@ -43,6 +43,8 @@ These tools were included in previous versions of Windows and the associated doc
...
+-   [Registry Editor](https://docs.microsoft.com/en-us/windows/win32/sysinfo/registry)
```
Comment (JohanFreelancer9) [inline windows/client-management/administrative-tools-in-windows-10.md]:
```suggestion
-   [Registry Editor](https://docs.microsoft.com/windows/win32/sysinfo/registry)
```

File: windows/client-management/administrative-tools-in-windows-10.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ The tools in the folder might vary depending on which edition of Windows you are
 
 ![Screenshot of folder of admin tools](images/admin-tools-folder.png)
 
-These tools were included in previous versions of Windows and the associated documentation for each tool should help you use these tools in Windows 10. The following list links to documentation for each tool.
+These tools were included in previous versions of Windows and the associated documentation for each tool should help you use these tools in Windows 10. The following list links to documentation for each tool. The tools are located within the folder C:\Windows\System32\ or its subfolders.
```
Comment (JohanFreelancer9) [inline windows/client-management/administrative-tools-in-windows-10.md]:
```suggestion
These tools were included in previous versions of Windows and the associated documentation for each tool should help you use these tools in Windows 10. The following list provides links to documentation for each tool. The tools are located within the folder C:\Windows\System32\ or its subfolders.
```

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
@Dansimp : This PR, based on issue ticket #5831, has been reviewed and is ready for final review & approval by the MS Docs team. If you yourself are too occupied with other projects, please assist us in assigning someone at your discretion, to follow up on this PR.

cc: @e0i , @mypil , @jvsam (not sure which one of you will be available in this case)

File: windows/client-management/administrative-tools-in-windows-10.md (hunk 4 above)
Comment (e0i) [reply windows/client-management/administrative-tools-in-windows-10.md]: @JohanFreelancer9 Is the folder path not wrapped around Markdown backticks?

**Comment by illfated** [comment]
GitHub Notifications indicated to me that there was supposed to be some kind of update in this PR. Yet, I don't see anything new here since February 11, 2020. Is there a hidden change here?

**Pull request merged**